cas/service.c

```diff
@@ -1326,6 +1330,24 @@ static int cas_fom_tick(struct m0_fom *fom0)
 			m0_ctg_op_init(&fom->cf_ctg_op, fom0,
 				       cas_op(fom0)->cg_flags);
 		}
+
+		/* If cas request has a txr payload, this a dtm0 operation.
+		 * We need to calculate credits for creating a dtm0 log record.
+		 *
+		 */
+		if (!m0_dtm0_tx_desc_is_none(&cas_op(fom0)->cg_txd)) {
+			struct m0_be_tx_credit dtm0logrec_cred;
```

Comment:
I wonder if credit calculation would work here - the structure was not initialized and it may have some random bytes in it. Was this code path covered in a UT?

Reply:
Thanks for catching this. This part of the code is in cas_fom_tick() and it is not covered in our UT today. 
I will fix this.